README.md

```diff
@@ -1,12 +1,7 @@
 # The Turing Way
 
 <!-- ALL-CONTRIBUTORS-BADGE:START - Do not remove or modify this section -->
-[![Build Status](https://img.shields.io/travis/alan-turing-institute/the-turing-way/master?logo=travis)](https://travis-ci.org/alan-turing-institute/the-turing-way)
```

Comment:
The bot has deleted a bit too much here! 🤖 

I think the tags marking the section that the bot can change were moved in a previous commit - I'll attempt to put it back here, but we'll have to make sure the bot doesn't do the same in its other already-generated PRs.